backend/mora/service/association.py

```diff
@@ -33,32 +33,35 @@ class AssociationRequestHandler(handlers.OrgFunkRequestHandler):
     def prepare_create(self, req):
         c = lora.Connector()
 
-        org_unit_uuid = util.get_mapping_uuid(req, mapping.ORG_UNIT,
-                                              required=True)
+        org_unit = util.checked_get(req, mapping.ORG_UNIT,
+                                    {}, required=True)
+        org_unit_uuid = util.get_uuid(org_unit, required=True)
 
         employee = util.checked_get(req, mapping.PERSON, {}, required=True)
         employee_uuid = util.get_uuid(employee, required=True)
 
-        org_unit = c.organisationenhed.get(org_unit_uuid)
+        org_unit_obj = c.organisationenhed.get(org_unit_uuid)
 
-        if not org_unit:
+        if not org_unit_obj:
             exceptions.ErrorCodes.E_ORG_UNIT_NOT_FOUND(
                 org_unit_uuid=org_unit_uuid,
             )
 
-        org_uuid = org_unit['relationer']['tilhoerer'][0]['uuid']
+        org_uuid = org_unit_obj['relationer']['tilhoerer'][0]['uuid']
         job_function_uuid = util.get_mapping_uuid(req, mapping.JOB_FUNCTION)
         association_type_uuid = util.get_mapping_uuid(req,
                                                       mapping.ASSOCIATION_TYPE,
                                                       required=True)
-        address_obj = util.checked_get(req, mapping.ADDRESS, {})
+
+        addr_id = util.get_mapping_uuid(req, mapping.ADDRESS)
```

Comment:
```suggestion
        function_id = util.get_mapping_uuid(req, mapping.ADDRESS)
```
DAR addresses have another UUID, so `addr_id` might be confusing.

Reply:
I've renamed it to `addr_function_id`